analyses/2019_03_agarwalishita_analysis.md

```diff
@@ -0,0 +1,30 @@
+The major difference between the dataset required to train/synthesize ad blocker and data blocker is described in this piece of text.
+
+**Ad blocker:**
+Its data mainly consists of filter rules which comes from EasyList, EasyPrivacy, etc. which consist of known ads on English Language websites all over the world. You can check the whole list [here](https://easylist.to/easylist/easylist.txt).
+
+  Ad blocker blocks data based on two things:
+
+1.  *Url domains*: Based on the dataset, ad blockers know which domains are not part of ad networks and those domains calls are blocked. You might have gotten idea about this idea after looking at the EasyList.
+    
+2.  *Tags name contained in your code*: It go through your script and see the tags, certain labelled tags as ads are blocked. In the same way, images under images/ads are blocked.
+    
+    So, the basic working of ad blocker can be seen as:
+
+-   URLs matched against filters.
+    
+-   DOM element names matched against element hiding filters.
+    
+-   Iframe content removed (iframe tag can be recognized by ad blocker).
+    
+-   Resource requests blocked.
+
+*Now question arises how tracker blocker like Ghost Rank used by ghostery browser is different from ad blocker. Does it uses the same technology? Are the datasets it is trained on is same as ad blocker?* Let’s talk about it in detail.
```

Comment:
* Reference / link to (a) Ghost Rank (b) Ghostery browser.
* I really don't think you can say "from ad blocker," I think you need to be specific on the type or implementation of ad blocker.
* Similarly, Ghostery just appears out of no-where here. What is the landscape of tracking protection, how does ghostery fit in?